packages/base/src/renderer/LitRenderer.ts

```diff
@@ -12,6 +11,40 @@ import type OpenUI5Enablement from "../features/OpenUI5Enablement.js";
 import type UI5Element from "../UI5Element.js";
 import { TemplateFunctionResult } from "./executeTemplate.js";
 
+type Renderer = (templateResult: TemplateFunctionResult, container: HTMLElement | DocumentFragment, styleStrOrHrefsArr: string | Array<string> | undefined, forStaticArea: boolean, options: RenderOptions) => void;
+
+type RenderOptions = {
```

Comment:
I think we should define in `RenderOptions` only the ones we pass (I think just `host`) and leave the others with their default values for lit specifically.

Reply:
done